src/components/views/AllTransactions.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+
+import PropTypes from 'prop-types';
+import { convertEthHelper } from '../../lib/helpers';
+import { feathersClient } from '../../lib/feathersClient';
+
+import User from '../../models/User';
+
+class AllTransactions extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      transactions: { data: [] },
+    };
+  }
+
+  componentWillMount() {
+    this.loadTransactions();
+  }
+
+  async loadTransactions() {
+    const currentUserAddress = this.props.currentUser.address;
+    console.log(currentUserAddress);
+    const transactions = await feathersClient.service('transactions').find({
+      query: {
+        address: currentUserAddress,
+      },
+    });
+
+    this.setState({ transactions });
+    console.log(this.state.transactions);
```

Comment:
Please remove as well.

Reply:
Does this mean you don't want the front end component called AllTransactions and that you're only interested in the back end PR?